### PR TITLE
Refactoring MultipleVFCreation_withMTU using new exec util:

### PR DESCRIPTION
**Pull request by dkosteck**
- Added a new util to run lists of commands and check exit codes in utils, cleaning up the test definitions.
- Implemented unit test for this new util.
- Refactored SR_IOV_MultipleVFCreation_withMTU using this new util as an example, will follow up with other tests in a new PR if this one is accepted

The goal of this PR is to demonstrate this new util, which is attempting to reduce duplicate code and make tests more concise. 

MultipleVFCreation_withMTU Report: https://gist.github.com/dkosteck/aac958a1b7547d9fba028428703fc6e6
test_utils Report: https://gist.github.com/dkosteck/c16e2c1f9adbd344ae91e2b6ad8ec911

### CODE REVIEW

File: sriov/common/utils.py [hunk 1]
```diff
@@ -223,9 +223,9 @@ def vfs_created(ssh_obj, pf_interface, num_vfs, timeout = 10):
     cmd = "ls -d /sys/class/net/" + pf_interface + "v* | wc -w"
     print(cmd)
     for i in range(timeout):
+        time.sleep(1)
```
Comment (jianzzha) [inline sriov/common/utils.py:226]: this line should already be on the main, did you rebase?

**Review by jianzzha — APPROVED** [review]

**Review by akiselev1 — APPROVED** [review]

File: sriov/common/utils.py (hunk 1 above)
Comment (jianzzha) [reply sriov/common/utils.py:226]: @dkosteck Did you rebase to solve the conflict?

Comment (dkosteck) [reply sriov/common/utils.py:226]: @jianzzha Not yet, will push an update shortly

**Comment by dkosteck** [comment]
@jianzzha Alright the merge conflict is resolved

**Pull request merged**